File: PHP/Exercicio05.php
<?php
echo "Informe seu salário atual: ";
$salarioAtual = fgets(STDIN); 

//1 Dólar dos Estados Unidos/USD (220) = 4,8931 Real/BRL (790)
$salarioAtualizado = $salarioAtual * (1+0.15);

echo "Salário atualizado: R$".number_format($salarioAtualizado, 2, ',', '.');

?>

File: Java/Exercicio05.java
package Java;

import java.util.Scanner;
import java.text.DecimalFormat;

public class Exercicio05 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Informe seu salário atual: ");
        double salarioAtual = scanner.nextDouble();

        double salarioAtualizado = salarioAtual * (1+0.15);
        DecimalFormat formatoMoeda = new DecimalFormat("R$ #,##0.00");
		System.out.println("Salário atualizado: "+ formatoMoeda.format(salarioAtualizado));

        scanner.close();
	}
}
